include/openrave/kinbody.h

```diff
@@ -991,6 +993,12 @@ class OPENRAVE_API KinBody : public InterfaceBase
         /// \brief returns true if the link is enabled. \see Enable
         bool IsEnabled() const;
 
+        /// \brief Ignores a Link from self collision.
+        void IgnoreSelfCollision(bool bIgnore);
```

Comment:
@kanbouchou let's call this `SetIgnoreSelfCollision`

Reply:
I changed the name from ``SetIgnoreSelfCollision``